test/e2e/specs/blocks/blocks__media.ts

```diff
@@ -12,13 +12,27 @@ import {
 	FileBlock,
 	TestFile,
 	TestAccount,
+	envVariables,
+	getTestAccountByFeature,
+	envToFeatureKey,
 } from '@automattic/calypso-e2e';
 import { Page, Browser } from 'playwright';
 import { TEST_IMAGE_PATH, TEST_AUDIO_PATH } from '../constants';
 
 declare const browser: Browser;
 
 describe( DataHelper.createSuiteTitle( 'Blocks: Media (Upload)' ), function () {
+	const features = envToFeatureKey( envVariables );
+	// @todo Does it make sense to create a `simpleSitePersonalPlanUserEdge` with GB edge?
+	// for now, it will pick up the default `gutenbergAtomicSiteEdgeUser` if edge is set.
```

Comment:
@worldomonation -- do you remember the reason behind the personal plan user? My gut is that this approach here should be just fine, but I wasn't involved in making that account.

Reply:
I wanted to have a user per WordPress.com plan tier to better separate tests. 
However, with the upcoming Plans reorganization it will no longer make sense, so I think we're good to disregard these users for now*.

* With the exception of simpleSitePersonalPlanUser that is being used frequently in specs. We can rename this user to something else that better reflects its position as a stand-in for `defaultUser`.